docs/usage/config-presets.md

```diff
@@ -14,6 +14,8 @@ In short:
 - Browse [Renovate's default presets](https://docs.renovatebot.com/presets-default/) to find any that are useful to you
 - Publish your own if you wish to reuse them across repositories
 
+UNSURE IF WANTED HERE: If you create a repository called `renovate-config` in your username namespace, Renovate finds this automatically and will suggest it in the `extends` field in new onboarding pull requests.
```

Comment:
We will also need to decide if we want to put a general explanation into `docs/usage/config-presets.md` or not?

Reply:
I think this should moved to onboarding config above and here only a reference to that.

Reply:
I've made a new commit that adds a placeholder draft link that we can fill in later.
I've removed the other explanations that were in the `docs/usage/config-presets.md` file.